Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -321,16 +247,139 @@ - (void)_stealFirstResponder;
 
 - (void)_showBugTitleCaptureAlert;
 {
-    UIAlertView *bugTitleCaptureAlert = [[UIAlertView alloc] initWithTitle:@"What Went Wrong?" message:@"Please briefly summarize the issue you just encountered. You’ll be asked for more details later." delegate:self cancelButtonTitle:@"Cancel" otherButtonTitles:@"Compose Report", nil];
-    bugTitleCaptureAlert.alertViewStyle = UIAlertViewStylePlainTextInput;
+    /*
+     iOS 8 often fails to transfer the keyboard from a focused text field to a UIAlertView's text field.
+     Transfer first responder to an invisble view when a debug screenshot is captured to make bug filing itself bug-free.
+     */
+    [self _stealFirstResponder];
     
-    UITextField *bugTitleTextField = [bugTitleCaptureAlert textFieldAtIndex:0];
-    bugTitleTextField.autocapitalizationType = UITextAutocapitalizationTypeSentences;
-    bugTitleTextField.autocorrectionType = UITextAutocorrectionTypeYes;
-    bugTitleTextField.spellCheckingType = UITextSpellCheckingTypeYes;
-    bugTitleTextField.returnKeyType = UIReturnKeyDone;
+    NSString *title = @"What Went Wrong?";
+    NSString *message = @"Please briefly summarize the issue you just encountered. You’ll be asked for more details later.";
+    NSString *okTitle = @"Compose Report";
+    NSString *cancelTitle = @"Cancel";
     
-    [bugTitleCaptureAlert show];
+    if ([UIAlertController class]) {
+        UIAlertController *alertController = [UIAlertController alertControllerWithTitle:title message:message preferredStyle:UIAlertControllerStyleAlert];
+        
+        [alertController addAction:[UIAlertAction actionWithTitle:okTitle style:UIAlertActionStyleDefault handler:^(UIAlertAction * _Nonnull action) {
+            UITextField *textfield = [alertController.textFields firstObject];
+            [self _shouldCreateReportWithTitle:textfield.text];
+        }]];
+        
+        [alertController addAction:[UIAlertAction actionWithTitle:cancelTitle style:UIAlertActionStyleDefault handler:NULL]];
+        
+        [alertController addTextFieldWithConfigurationHandler:^(UITextField * _Nonnull textField) {
```

Comment:
We'll need to put this code under `#ifdef __IPHONE_9_0` so it'll compile on Xcode 6.3-6.4